src/main/java/Command.java

```diff
@@ -0,0 +1,68 @@
+public class Command {
+    private String commandType;
+
+    public Command(String type) {
+        this.commandType = type;
+    }
+
+    public void execute(TaskList tasklist) {
```

Comment:
this method is a little long. perhaps there is a better way of parsing the user input?

Reply:
might also find a better home in the `Parser` class.